tsconfig.other.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
```

Comment:
do you think we could extend `manifest.json` as well? I think it checks if file exists on runtime, and routed options page cannot be accessed then. 
But again I think it is impossible to use react-router anyways, as routing is done on the client side, not on the server side. Food for thought.

Reply:
There isn't anything regarding inheritance in https://developer.chrome.com/extensions/manifest, while there is in https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends.